structural-variantcalling.wdl

```diff
@@ -276,3 +285,32 @@ workflow SVcalling {
                            category: "advanced"}
     }
 }
+
+
+task CleverWorkaround {
+    input {
+        File vcf
+        String outputPath
+    }
+
+    command {
+        set -e
+        mkdir -p $(dirname ~{outputPath})
+        sed 's/\tSVLEN=/\tWORKAROUND;SVLEN=/' ~{vcf} > ~{outputPath}
```

Comment:
Won't this break the INFO field specification? Where the number of info fields is fixed?

Reply:
INFO doesn't have a fixed number of fields. Info field can be flags (present or absent indicating a Boolean value). Technically speaking, a line should be added in the header describing the INFO field. But since this VCF only gets used by survivor that doesn't matter. Survivor has it's own VCF parser implementation which ignores the header.

Reply:
Thanks for the explanation. Can you put that in a comment above this line of code? After that it is good to go as far as I am concerned.